src/RunProcessAsTask/RunProcessAsTask.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netstandard1.4;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.4;netstandard2.0;</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">net45;$(TargetFrameworks)</TargetFrameworks>

```

Comment:
On which OS is the nuget package built?
Do we need to be this clever?

Reply:
Otherwise, it won't build on unix systems (travis ci)

Reply:
Better to have test coverage on Unix systems. There're gaps between unix and windows around Process APIs, e.g.  https://github.com/dotnet/corefx/issues/35544